include/tha.h

```diff
@@ -0,0 +1,28 @@
+#ifndef THA_H
+#define THA_H
+
+#include "ultra64.h"
+#include "libc/stdint.h"
+
+typedef struct TwoHeadArena {
+    /* 0x00 */ size_t size;
+    /* 0x04 */ void* start;
+    /* 0x08 */ void* head;
+    /* 0x0C */ void* tail;
+} TwoHeadArena; // size = 0x10
```

Comment:
```suggestion
    /* 0x0 */ size_t size;
    /* 0x4 */ void* start;
    /* 0x8 */ void* head;
    /* 0xC */ void* tail;
} TwoHeadArena; // size = 0x10
```
I think we decided to pad to the number of digits required within the struct itself rather than including the size in that.
https://github.com/zeldaret/mm/pull/1143#discussion_r1021067301